docs/reference/connectors/udp.md

```diff
@@ -7,6 +7,12 @@ sidebar_position: 1
 
 The `udp` connector allows UDP based datagram clients and servers to be integrated with tremor.
 
+The `udp` client, by default binds to `0.0.0.0:0` allowing to send to all interfaces of the system running tremor and picking a random port. This can be overwritten adding `"bind": "<ip>:<port>"` to the `config`.
+
+:::warn
+If you are hardening a installation it might make sense to limit the interfaces a udp client can send to by specifying the `"bind"` config.
```

Comment:
```suggestion
If you are hardening an installation it might make sense to limit the interfaces a udp client can send to by specifying the `"bind"` config.
```